docs/api/pet/delete_pet.md

```diff
@@ -5,3 +5,53 @@ nav_order: 5
 ---
 
 # Delete pet
+
+**DELETE** `/pets/{id}`
+
+Delete the [`pet`](index.md) in the service with the specified `id`.
+
+## URL
+
+```shell
+{base_url}/pets/{id}
+```
+
+### Path Parameters
+
+| Parameter name | Type | Description |
+| -------------- | ---- | ----------- |
+| `id` | number | The `id` of the [`pet`](index.md#resource-properties) to delete. |
+
+### Query Parameters
+
+None
+
+## Request
+
+### Request headers
+
+None
+
+### Request body
+
+None
+
+## Response
+
+### Response body
+
+Returns an empty object.
```

Comment:
You could say "returns an empty JSON object" here, for precision.

Reply:
Updated throughout!